src/ActionModal/components/ActionModal.vue

```diff
@@ -728,41 +723,32 @@ export default {
 
       this.trackEvent(`event`, `submit`, this.title, this.selectedSignMethod)
 
-      const { type, memo, ...properties } = this.transactionData
-      const feeProperties = {
-        gasEstimate: this.gasEstimate,
-        gasPrice: {
-          // the cosmos-api lib uses gasEstimate * gasPrice to calculate the fees
-          // here we just reverse this calculation to get the same fees as displayed
-          amount: this.estimatedFee / this.gasEstimate,
-          denom: this.getDenom
-        },
-        submitType: this.selectedSignMethod,
-        password: this.password
-      }
-      const txMetaData = {
-        ...feeProperties,
-        displayedProperties: {
-          claimableRewards: properties.amounts
-        }
-      }
+      const { type, memo, ...message } = this.transactionData
 
       try {
-        await this.$apollo.queries.overview.refetch()
-        const hashResult = await this.actionManager.sendTxAPI(
-          {
-            network: this.network,
-            networkType: this.network.network_type,
-            chainId: this.network.chain_id,
-            userAddress: this.session.address,
-            rewards: this.rewards,
-            account: this.overview.accountInformation
-          },
-          type,
-          memo,
-          properties,
-          txMetaData
-        )
+        let transactionData
```

Comment:
@bitcoinera here you see the simplified flow in the ActionModal

Reply:
nice